sdk/core/core-lro/package.json

```diff
@@ -142,7 +140,6 @@
     "rollup-plugin-sourcemaps": "^0.4.2",
     "rollup-plugin-terser": "^5.1.1",
     "rollup-plugin-visualizer": "^3.1.1",
-    "shx": "^0.3.2",
```

Comment:
I'm not even sure why this dependency is here. Tests are passing just as usual without these.